src/Pages/PeoplePage.tsx

```diff
@@ -0,0 +1,84 @@
+import { useCallback, useEffect, useState } from 'react';
+import { Loader } from '../components/Loader';
+import { Person } from '../types';
+import { getPeople } from '../api';
+import { PeopleTable } from './PeopleTable';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoaded, setIsLoaded] = useState<boolean>(true);
+  const [isError, setIsError] = useState<boolean>(false);
+
+  const fetchPeople = useCallback(async () => {
+    setIsLoaded(false);
+    try {
+      const inData = await getPeople();
+
+      setPeople(inData);
+    } catch (inError) {
+      setIsError(true);
+    } finally {
+      setIsLoaded(true);
+    }
+  }, []);
+
+  useEffect(() => {
+    fetchPeople();
+  }, []);
+
+  return (
+    <>
+      <h1 className="title">People Page</h1>
+      <div className="block">
+        <div className="box table-container">
+          {!isLoaded
+            ? <Loader />
+            : (
+              <>
+                {isError && (
+                  <p data-cy="peopleLoadingError" className="has-text-danger">
+                    Something went wrong
+                  </p>
+                )}
+
+                {people.length <= 0 && (
```

Comment:
less or equal zero? what? how can array.length be less then zero?)

Reply:
magic) thanks)